components/Icon/Icon.jsx

```diff
@@ -0,0 +1,20 @@
+import Image from 'next/image'
+import menu from '../../public/images/menu-icon.svg'
+import close from '../../public/images/close-icon.svg'
+import arrowUp from '../../public/images/arrow-up-icon.svg'
+
+const icons = {
+    menu,
+    close,
+    arrowUp,
+}
+
+const Icon = (props) => {
+    const { variant, width, height } = props
+    const size = {}
```

Comment:
This could be: 
`size = { width, height }` 

No need to check if its exist

Reply:
Done